installation.html

```diff
@@ -126,6 +126,30 @@ <h3 id="editor">Editor Configuration</h3>
 <li><a href="configuration-texshop.html"><span class="tex">T<span class="epsilon">e</span>X</span>Shop</a> (a Mac only editor)</li>
 </ul>
 
+<h3 id="uninstall">Uninstalling Gregorio</h3>
```

Comment:
I think you should make a particular note that these uninstall options are only if you manually installed Gregorio.  If it was installed using TeX Live or a package manager, then it should be uninstalled using same.

Reply:
Umm... I think I already did that.  Is what I wrote below [line 132 and on](https://github.com/gregorio-project/gregorio-project.github.io/pull/85/files#diff-518ca5fb7da42aa3de8d6009c13c1ac7R132) not clear on that point?  Any suggestions for improving it?

Reply:
OK, not sure how I missed it, but you're right. This should be fine.